lib/consumer/data_updater/handler.ex

```diff
@@ -37,6 +48,13 @@ defmodule Mississippi.Consumer.DataUpdater.Handler do
   @callback handle_signal(signal :: term(), state :: handler_state) ::
               {result :: term(), new_state :: handler_state}
 
+  @doc """
+  Invoked when handle_message returns a `{:continue, continue_arg}` tuple.
+  Used to update the state of a stateful handler.
+  """
+  @callback handle_continue(continue_arg :: term(), state :: handler_state) ::
+              new_state :: handler_state
```

Comment:
With the changes introduced in this pr we now allow for the following workflow:
message handle -> ack/discard -> nothing/continue/stop

This definition does not give you the expressive power to configure the following behavior:
message handle -> ack/discard -> do something else (continue) -> stop

Even if we don't want to handle this situation right now I still think this is a potentially valid workflow, so I suggest we change the return value to eg `{:ok, new_state}` so that in the future we can add eg `{:stop, new_state}` without making a breaking change

```suggestion
  @callback handle_continue(continue_arg :: term(), state :: handler_state) ::
              {:ok, new_state :: handler_state}
```

Reply:
Good point! Done